test/tests/kernels/curlcurl.i

```diff
@@ -62,10 +62,6 @@
     type = MFEMVectorFunctionCoefficient
     function = exact_e_field
```

Comment:
Function Dirichlet BCs can also now take the function name directly, now `addFunction` automatically sets the coefficient (but assuming that's planned for a later PR when `MFEMCoefficients` are removed?)

Reply:
Yes, I'm leaving that for the next PR (which I'll probably open tomorrow).